src/lib.rs

```diff
@@ -1,6 +1,6 @@
 //! A ledger for memory mappings.
 
-#![cfg_attr(not(test), no_std)]
+//#![cfg_attr(not(test), no_std)]
```

Comment:
Don't. :)

Reply:
a mistake (had some temporary prints). thanks for spotting that out and apologies.